src/duration.rs

```diff
@@ -285,7 +285,7 @@ impl fmt::Display for FormattedDuration {
         let micros = nanos / 1000 % 1000;
         let nanosec = nanos % 1000;
 
-        let ref mut started = false;
+        let started = &mut false;
```

Comment:
I find the old variant more useful. This looks like a mutable reference to the constant. I understand this is equivalent, but less readable in my opinion.

Reply:
It's just the result of running the command `cargo + nightly fix -Z unstable-options --clippy --allow-dirty`.
If you don't like it, I will manually revert it.

Reply:
Yes, please. The clippy is not perfect. And readability is what matters more than clean clippy output. This crate is probably going to be not changed for years from now, so we don't want to update library if clippy changes. And we don't have so large number of contributions to make it justifiable.